tests/integration/models/reddit/test_wikipage.py

```diff
@@ -72,6 +84,16 @@ def test_revisions(self, _):
                 assert isinstance(revision['page'], WikiPage)
             assert count > 0
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_revisions__author_deleted(self, _):
+        subreddit = self.reddit.subreddit(
+            pytest.placeholders.test_subreddit)
+
+        with self.recorder.use_cassette(
+                'TestWikiPage.test_revisions__author_deleted'):
+            revisions = subreddit.wiki['index'].revisions(limit=10)
+            assert len(list(revisions)) == 10
```

Comment:
I would recommend doing something like:

    assert any(revision.author is None for revision in revisions)